zerver/context_processors.py

```diff
@@ -63,6 +63,7 @@ def add_settings(request):
         'find_team_link_disabled': settings.FIND_TEAM_LINK_DISABLED,
         'password_min_length': settings.PASSWORD_MIN_LENGTH,
         'password_min_quality': settings.PASSWORD_MIN_ZXCVBN_QUALITY,
+        'zulip_version': ZULIP_VERSION
```

Comment:
put a trailing comma here; it prevents us from having to change the previous line when adding something to a multi-line dict like this